testing/A_introduction.md

```diff
@@ -41,10 +37,11 @@ test
 ├── test_helper.exs
 └── views
     ├── error_view_test.exs
+    ├── layout_view_test.exs
```

Comment:
This is funny. I actually wrote the PR in Phoenix to put this file in the generated application, but I didn't fix the guide. Derp. ;^)

Thanks for catching my mistake!